docs/services/gcp-services-guide.md

```diff
@@ -0,0 +1,620 @@
+---
+title: "Using GCP Services"
+toc: true
+weight: 420
+indent: true
+---
+
+# Deploying Wordpress to GKE
```

Comment:
I believe this should also be "GCP Service Guide" for consistency across services guides.